### PR TITLE
✨  Add new tokenomics UI

### PR DESCRIPTION
**Pull request by MarioBatalha**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206456889270869

### CODE REVIEW

**Review by KipandaJr — CHANGES_REQUESTED** [review]
In general the this 
image
 dosent look equal that figma 
image

File: components/svg/dex-screener.tsx [hunk 1]
```diff
@@ -4,29 +4,37 @@ import { SVGProps } from './svg.types';
 
 const DexScreener: FC<SVGProps> = ({ maxHeight, maxWidth, ...props }) => (
```
Comment (git-marcopitra) [inline components/svg/dex-screener.tsx:5]: this changes are out of the scope, and doesn't make sense

**Review by git-marcopitra — CHANGES_REQUESTED** [review]

**Review by KipandaJr — CHANGES_REQUESTED** [review]
image

**Review by git-marcopitra — APPROVED** [review]

File: components/layout/footer/index.tsx [hunk 2]
```diff
@@ -23,7 +23,8 @@ const Footer: FC<LayoutProps> = ({ withoutRoadmap }) => {
   return (
     <Box
       width="100%"
-      background="linear-gradient(180deg, #DC77F7 19.58%, #7B2FC4 149.72%)"
+      background="linear-gradient(180deg, #DB76F6 0%, #451779 100%);
+      "
```
Comment (git-marcopitra) [inline components/layout/footer/index.tsx:27]: why do we have this `;`

File: views/home/index.tsx [hunk 3]
```diff
@@ -7,7 +7,7 @@ import AboutSection from './about';
 import Advisor from './advisor';
 import AsSeenOn from './as-seen-on';
 import Collaborators from './collaborations';
-import DisclaimerSection from './disclaimer';
+import DisclaimerSection from './disclaimer-section';
```
Comment (git-marcopitra) [inline views/home/index.tsx:10]: don't use the suffix `Section` on files, neither on the components

**Review by git-marcopitra — CHANGES_REQUESTED** [review]

**Review by atilasousa — APPROVED** [review]

**Review by git-marcopitra — APPROVED** [review]

**Pull request merged**